update-po-files.sh

```diff
@@ -63,6 +63,7 @@ cd $SOURCEDIR
 
 #Generate/update template pot file
 find $POTDIRS -name '*.cpp' -or -name '*.h' |
+  sort |
```

Comment:
People on stackexcange use the -z option to sort when sorting results from find, I am not sure if that makes sense.